README.md

```diff
@@ -52,3 +51,5 @@ Qt creator should now format the documents when saving it.
 
 ## Doxygen
 Doxygen is used to generate project information daily and is available [here](https://doxygen.chatterino.com).
+## chocolatey 
+ A chocolatey version is available [here](https://chocolatey.org/packages/chatterino).  
```

Comment:
The chocolatey section should be next to the Download section, not at the bottom next to things it isn't related to.